scala/private/rule_impls.bzl

```diff
@@ -355,28 +322,28 @@ def _compile_or_empty(ctx, manifest, jars, srcjars, buildijar,
         f for f in ctx.files.srcs if f.basename.endswith(_scala_extension)
     ] + java_srcs
     compile_scala(ctx, ctx.label, ctx.outputs.jar, manifest,
-                  ctx.outputs.statsfile, sources, jars, all_srcjars, buildijar,
+                  ctx.outputs.statsfile, sources, jars, all_srcjars,
                   transitive_compile_jars, ctx.attr.plugins,
                   ctx.attr.resource_strip_prefix, ctx.files.resources,
                   ctx.files.resource_jars, jars2labels, ctx.attr.scalacopts,
                   ctx.attr.print_compile_time, ctx.attr.expect_java_output,
                   ctx.attr.scalac_jvm_flags)
-    # compile the java now
-    if buildijar:
-      scala_output = ctx.outputs.ijar
-    else:
-      scala_output = ctx.outputs.jar
-    java_jar = try_to_compile_java_jar(
-        ctx, scala_output, all_srcjars, java_srcs,
-        implicit_junit_deps_needed_for_java_compilation)
 
-    ijar = None
+    # compile the java now
```

Comment:
this comment from the diff looks to be around the ijar rather than the java?

Reply:
well, yeah, it is before the ijar which is before java. Due to CI slowness, I'll pick this up in the next PR (the merge of #524 with this).